tsconfig.json

```diff
@@ -5,8 +5,8 @@
     "esModuleInterop": true,
     "incremental": true,
     "lib": ["es2015", "dom"],
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "module": "Node16",
+    "moduleResolution": "Node16",
```

Comment:
This does not yet signal dropping Node.js 14, which is scheduled in https://github.com/aws/aws-sdk-js-v3/issues/5416.

This is needed for the TypeScript compiler to understand the `exports` block of the `@aws-sdk/external` package. In Node.js this was added in v12 already.

I will test to confirm this behavior.